doc/api/deprecations.md

```diff
@@ -3145,6 +3145,20 @@ parameter in [`fs.write()`][], [`fs.writeFile()`][], [`fs.appendFile()`][],
 [`fs.writeFileSync()`][], and [`fs.appendFileSync()`][] is deprecated.
 Convert them to primitive strings.
 
+### DEP0163: `process.exit([code])` coercion to integer
+
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/43738
+    description: Documentation-only deprecation.
+-->
+
+Type: Documentation-only
+
+Implicit coercion of a non-integer `code` parameter in [`process.exit()`][] is
+deprecated. Please use an integer value.
```

Comment:
```suggestion
`code` values other than `undefined`, `null`, integer numbers and integer
strings are deprected as parameter in [`process.exit()`][].
```

Reply:
Is using string a common thing? We usually enforce the type strictly in most of our functions, it would make sense to do that here (unless it'd be too breaking?)

Reply:
Changed according to the review.